### PR TITLE
Fix price point decimals accuracy

### PR DESCRIPTION
**Pull request by hyphenized**
Previously if we had two assets with the same symbol and different decimals count on different networks, we would calculate balances incorrectly

## To Test
- [ ] Go to etherscan and bsc, balances should match estimates shown in the wallet main and overview pages

Latest build: [extension-builds-2711](https://github.com/tallycash/extension/suites/9735237297/artifacts/467119752) (as of Wed, 07 Dec 2022 23:51:53 GMT).

### CODE REVIEW

**Review by jagodarybacka — COMMENTED** [review]
It seems to be working correctly but it would be nice to check it on BNB - could you fix the conflicts so we will have BNB available on this branch 😅

File: background/redux-slices/assets.ts [hunk 1]
```diff
@@ -198,44 +193,54 @@ export const transferAsset = createBackgroundAsyncThunk(
...
 
     if (pricedAsset) {
-      const pricePoint = pricedAsset.recentPrices[pairedAssetSymbol]
-      const { pair, amounts, time } = pricePoint
-
-      if (pair[0].symbol === assetSymbol) {
-        return pricePoint
+      let pricePoint = pricedAsset.recentPrices[pairedAssetSymbol]
+
+      // Flip it if the price point looks like USD-ETH
```
Comment (0xDaedalus) [inline background/redux-slices/assets.ts:211]: Can we add some context here as to why we flip?

File: background/redux-slices/assets.ts [hunk 2]
```diff
@@ -198,44 +193,54 @@ export const transferAsset = createBackgroundAsyncThunk(
...
-    const pricedAsset = (
-      chainID
-        ? assets.filter(
-            (asset) =>
-              "homeNetwork" in asset && asset.homeNetwork.chainID === chainID
-          )
-        : assets
-    ).find(
+  [selectAssetsState, selectAsset, selectPairedAssetSymbol],
+  (assets, assetToFind, pairedAssetSymbol) => {
```
Comment (0xDaedalus) [inline background/redux-slices/assets.ts:197]: Non-blocking but can we follow up this PR with some tests please?

**Review by 0xDaedalus — COMMENTED** [review]
Code looks good - will test manually to confirm

File: background/redux-slices/selectors/accountsSelectors.ts [hunk 3]
```diff
@@ -82,8 +82,7 @@ const computeCombinedAssetAmountsData = (
   assets: AssetsState,
   mainCurrencySymbol: string,
   currentNetwork: EVMNetwork,
-  hideDust: boolean,
-  useCurrentNetwork = false
```
Comment (greg-nagy) [inline background/redux-slices/selectors/accountsSelectors.ts:86]: 🎉

**Review by 0xDaedalus — APPROVED** [review]
🚀

**Review by greg-nagy — APPROVED** [review]
let's not block this one now, but please have tests next time :) 

We should get into testing every PR

File: background/redux-slices/assets.ts (hunk 2 above)
Comment (0xDaedalus) [reply background/redux-slices/assets.ts:197]: @hyphenized would you mind creating a follow-up issue here and assigning it to yourself?

**Pull request merged**